app/views/foreman_ansible/ansible_roles/_select_tab_content.html.erb

```diff
@@ -2,7 +2,7 @@
 <%= webpacked_plugins_css_for :foreman_ansible %>
 
 <div class='tab-pane' id='ansible_roles'>
-  <% roles = f.object.is_a?(Hostgroup) ? roles_attrs(f.object.inherited_and_own_ansible_roles) : roles_attrs(f.object.all_ansible_roles) %>
+  <% roles = role_attributes_for_roles_switcher(f.object) %>
```

Comment:
given we use it just once, it probably make sense to use it directly, instead of assigning the variable and using the variable. WDYT?